### ISSUE TITLE
webMAN not ready

### ISSUE THREAD

**Issue by hydrohs** [aldostools/webMAN-MOD#1014]
I cannot get webMAN to start. I have reinstalled multiple times, as well as (manually, following directions on the wiki) uninstalled several times to no avail. Had the issue with Rebug 4.84 as well as currently Evilnat 4.90. Any time I try to perform any action I get the message "webMAN not ready", anything that opens in the built-in browser is just a blank page, error 8071053d.

Networking is enabled and I can access the PS3 over the Cobra FTP server, as well as access sites on the internet from the PS3.

**Comment by aldostools** [comment]
Please confirm that Cobra is enabled and that /dev_hdd0/boot_plugins.txt exists and that it has the correct path of the plugin.

If you enter to recovery menu, boot_plugins.txt is deleted.

If you switched to DEX mode, the plugin should be loaded using /dev_hdd0/boot_plugins_dex.txt

The message that you get usually occurs when the plugin was not loaded

**Comment by hydrohs** [comment]
Cobra is enabled, and /dev_hdd0/boot_plugins.txt exists and has /dev_hdd0/plugins/webftp_server.sprx as its contents. This file does exist, and I am running on CEX mode.

**Comment by aldostools** [comment]
Can you load other plugins? Or are other plugins running that could cause conflicts?

Error 0x8071053D = Server timed out. That means that webMAN MOD is not loaded (or the web server could not be initialized).

An easy way to verify if webMAN MOD is running is pressing SELECT+START. It should show a popup with the system information.

**Comment by hydrohs** [comment]
Pressing Select + Start does nothing, I have no other plugins loaded. I'd be happy to try some others but I have no clue what other plugins exist that I could test.

**Comment by aldostools** [comment]
There are various plugins included in webMAN MOD.

You can try installing manually VshFtpCounter to check if plugins are loading
https://github.com/TheRouletteBoi/RouLetteVshMenu/releases/tag/fps-counter-2.7

**Comment by hydrohs** [comment]
Okay, that worked no problem.

**Comment by aldostools** [comment]
Then some setting or external device could be preventing that webMAN MOD load properly.

- Try deleting /dev_hdd0/tmp/wm_config.bin to reset the settings to the default.
- Make sure that you have installed the latest webMAN MOD 1.47.45 on Evilnat 4.90
- Disconnect ALL USB devices (controller, external disk, pendrives, etc.)

If webMAN MOD loads, go to /setup.ps3 and check the option "Disable scan for content at startup". Save and reboot.

**Comment by hydrohs** [comment]
Unfortunately that made no difference

**Comment by KTMGv5** [comment]
I fixed my issues following this post on this thread and aldostools commented the Information you will need. [https://www.psx-place.com/threads/how-to-uninstall-webman-mod.28521/](url) After delteting the files restart your PS3 and reinstall webMAN MOD from the installer holding L1 after pressing Cross on the installer. DO not DELETE your GAME folder 👍 i spent 6 hours following the internet loophole trust the process if all else fails check and see if Sys 8 is loading ? 💯

**Comment by hydrohs** [comment]
Alright, I thought I would try replacing the hard drive and starting totally from scratch, but that failed. And upon reinstalling the old drive and starting up, webMAN loaded. "Disable scan for content at startup" was already checked, and after rebooting I'm back to having the same issue. Not sure if that helps or makes things more confusing.

**Comment by hydrohs** [comment]
I have this working now. The wireless module in my console was not working. Once I repaired it, webMAN started working again. Is it depending on Wifi/Bluetooth for something?

**Comment by aldostools** [comment]
Yes, webMAN MOD is a network server plugin for HTTP, FTP, NET (ps3netsrv) and ps3mapi.

It uses network for many functions. e.g. Mount games, download files, setup configuration, etc.

**Comment by hydrohs** [comment]
But does it specifically require a working Wifi module? I had, and still have, my console wired. I am not using Wifi at all, the only difference now is that if I wanted to I could. However it's not connected to any networks via wireless.

**Comment by aldostools** [comment]
I only have tested webMAN MOD on fully working PS3 systems.

The system driver that manages the network communication could have been affected by the malfunction of the Wifi module.

**Comment by MatheusDsouza** [comment]
Hello guys! Any solution besides repairing the module? I have a similar problem, but, if I press STAR+SELECT shows the popup with the system information, but if I try to open the config shows the error "8071053d" and if I try to open sLaunch or even try to uninstall shows the popup with "webMAN not ready". Sadly I can't fix the problem with the module because the local technical assistances doesn't have the module and they said that there's no expected arrival date for the module.

System info:
PS3 Slim CECH 25xx
Evilnat 4.91.2 CEX [no bt]

**Comment by aldostools** [comment]
Try installing the latest release.
The recent builds have a new version of wm_proxy that mounts the games directly without using the network interface. It could be helpful when the BT/WiFi module is not working.

If that doesn't work, then I don't have other solution, due webMAN-MOD relies on http.

To refresh the game list use the combos.

**Comment by MatheusDsouza** [comment]
I'm using the latest version, I can mount games but I can't access the configs, and it's a problem cause I need to make some configs at my fan, and I can't access the configs remotly either

**Comment by aldostools** [comment]
The webMAN's configuration page is HTML-based. Without access to the web server is not possible to edit the settings.

The unique solution is to edit the `wm_config.bin` found in /dev_hdd0/tmp with an hex editor.

If someone create a Windows application or Python script to edit the file, it could be modified in a PC and uploaded to the PS3.

Another option is that someone edit the settings for you on another PS3 and send you his/her `wm_config.bin` file.

**Comment by MatheusDsouza** [comment]
The console is connected via ethernet cable, so internet works, but it seens that the webMAN needs the wi-fi module working

**Comment by aldostools** [comment]
It looks like the socket library fails if the BT/Wi-Fi module is not working.

Without a broken PS3 console it's hard for me to test alternatives to make it working without Wi-Fi.

The best solutioni is that you repair your PS3. The module is not expensive and can be found in eBay.

Here are a couple of tutorials that I found after a quick search on Google.
https://www.youtube.com/watch?v=gXfF5Py_bEQ
https://www.youtube.com/watch?v=nmFa1LNNwFE

**Comment by MatheusDsouza** [comment]
UPDATE:

Well, I changed my internet provider and now webMan is working as expected, dunno what happens but now it's okay

**Comment by MatheusDsouza** [comment]
Sad UPDATE:

Don't know what happen, I changed the storage to a SSD and the error returned

**Comment by MatheusDsouza** [comment]
Maybe multiMan and webMan are having a incompatibility? I remember that when I'm shearching for ways to resolve that problem, in a post you mentioned that if webMan and multiMan are using the same port we will have issues, but I can't find this post anymore

**Comment by MatheusDsouza** [comment]
Maybe is some internet configuration, I have done a quick format and installed only webMAN, but the issue persists

**Comment by aldostools** [comment]
Latest official MultiMAN and webMAN MOD are incompatible.

Your issue could be related to this.

multiMAN MOD is a modification that I made to resolve the conflict and add support for exFAT.

Get it from http://store.brewology.com/ahomebrew.php?brewid=24

**Comment by MatheusDsouza** [comment]
I don't think so, I'm using multiMAN MOD + webMAN MOD

**Comment by aldostools** [comment]
Well, you asked if there was any incompatibility and I answered that there was one.

If it's not software incompatibility, then your issue can to be hardware related.

**Comment by Joonie86** [comment]
No further feedback received.
